guides/common/modules/con_overview-of-recommended-disaster-recovery-plans.adoc

```diff
@@ -0,0 +1,80 @@
+[id="overview-of-recommended-disaster-recovery-plans"]
+= Overview of recommended disaster recovery plans
+
+Choose a disaster recovery plan that best helps ensure the continuity of {Project} services in your deployment.
+
+//[IMPORTANT]
+//====
+//During recovery, you will need to ensure that the hostname of your {ProjectServer} does not change.
+//The IP address can change.
+//====
+
+Virtualizing your {ProjectServer}::
```

Comment:
Some virtualization products like VMWare provide built in tools to move and restart VM if they die. How should we include recommendation of using such tools if their hypervisor provides it?

Reply:
I think we shouldn't. It's generally not a good idea to document features and workflows associated with third-party products: we can't commit to keeping up with how they evolve, we can't provide assurance to users that they work, and we can't provide alternative solutions in case they don't work.

Let's see if this comes up again in one of the follow-up PRs for the more specific steps; if so, we can discuss further, perhaps adding a link to third-party docs could work.